docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.5'
+
+services:
+
+  nhsd.admin:
+    image: nhsd/buying-catalogue/admin
+    container_name: nhsd_bc_admin
+    build:
+      context: .
+      dockerfile: Dockerfile
```

Comment:
what about NODE_ENV?

Reply:
that's 'injected' into it by the orchestrator (your local docker-compose / local k8s or in prod aks)

Reply:
if 'development' is injected it will break on browser-sync

Reply:
I think its redundant, but I guess it doesn't hurt to have it there just in case the orchestrator doesn't set it.